test/suites/dev-frontier-template/test-pov/test-evm-over-pov.ts

```diff
@@ -0,0 +1,107 @@
+import "@moonbeam-network/api-augment";
+import { beforeAll, deployCreateCompiledContract, describeSuite, expect } from "@moonwall/cli";
+import { ALITH_ADDRESS, createEthersTransaction } from "@moonwall/util";
+import { Abi, encodeFunctionData } from "viem";
+import { expectEVMResult, HeavyContract, deployHeavyContracts } from "../../../helpers";
+
+describeSuite({
+    id: "DF1301",
+    title: "PoV controlled by gasLimit",
+    foundationMethods: "dev",
+    testCases: ({ context, it, log }) => {
+        let proxyAddress: `0x${string}`;
+        let proxyAbi: Abi;
+        let contracts: HeavyContract[];
+        let callData: `0x${string}`;
+        const MAX_CONTRACTS = 20;
+        const EXPECTED_POV_ROUGH = 38_000; // bytes
```

Comment:
same thing with this, why is our expected pov different than moonbeam one?

Reply:
Our tests seems to fail with the original values. Reverted them in https://github.com/moondance-labs/tanssi/pull/716/commits/58db5bd6992983f6a2df28a7ee03b2569cc69a90 to new values and will be looking into resolving the issue now

Reply:
Reverted to Moonbeam's values before the recent Frontier update in https://github.com/moondance-labs/tanssi/pull/716/commits/e8dfc8d9c0c3b23570326143b38b5d72e3b2a742